spark/v3.4/spark/src/jmh/java/org/apache/iceberg/spark/data/parquet/SparkParquetReadersFlatDataBenchmark.java

```diff
@@ -156,6 +156,8 @@ public void readUsingSparkReader(Blackhole blackhole) throws IOException {
             .set("spark.sql.parquet.int96AsTimestamp", "false")
             .set("spark.sql.caseSensitive", "false")
             .set("spark.sql.parquet.fieldId.write.enabled", "false")
+            .set("spark.sql.parquet.inferTimestampNTZ.enabled", "false")
```

Comment:
Any hint why we need these new params?

Reply:
I guess we are reading a file not written by Spark so there is no Spark schema written. That's why Spark tries to infer the actual timestamp type.

Reply:
This is due to these 2 flags being required: https://github.com/apache/spark/blame/branch-3.4/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetSchemaConverter.scala#L72-L73